kitsune/wiki/models.py

```diff
@@ -1055,6 +1063,15 @@ def get_mapping_type(cls):
         return RevisionMetricsMappingType
 
 
+class DraftRevision(ModelBase, SearchMixin, AbstractRevision):
+    based_on = models.ForeignKey(Revision, blank=True, null=True)
+    content = models.TextField(blank=True)
+    locale = LocaleField(default=settings.WIKI_DEFAULT_LANGUAGE)
```

Comment:
As the document will always be a ``en-US`` one, we can not. We identify with parent document, and locale